File: spec/python/test_default_big_endian.py
import unittest

from default_big_endian import DefaultBigEndian

class TestDefaultBigEndian(unittest.TestCase):
    def test_default_big_endian(self):
        with DefaultBigEndian.from_file("src/enum_0.bin") as r:

            self.assertEqual(r.one, 0x7000000)

File: spec/lua/test_default_big_endian.lua
local luaunit = require("luaunit")

require("default_big_endian")

TestDefaultBigEndian = {}

function TestDefaultBigEndian:test_default_big_endian()
    local r = DefaultBigEndian:from_file("src/enum_0.bin")

    luaunit.assertEquals(r.one, 0x7000000)
end
